docs/server/commands.rst

```diff
@@ -901,6 +891,16 @@ run_cherrypy
 Run the CherryPy server bundled with the Translate Toolkit.
 
 
+.. django-admin:: start
+
+start
+^^^^^
+
+.. deprecated:: 2.7.3
```

Comment:
No they shouldn't just be removed from docs.  We're not doing any deprecation phase and we don't want to carry this around with us following some deprecation strategy.  But we do want to communicate to users that the command is gone and what they should do without it.  That requires it in the docs.  Deprecated works TBH, but you got a new directive

And if any of you don't make use of it may your PRs itch